<launch>
    <arg name="project" default="lvi_sam" />

	<param name="/use_sim_time" value="true" />

	<!-- Visual feature and odometry -->
	<node pkg="$(arg project)" type="$(arg project)_visual_feature" name="$(arg project)_visual_feature" output="screen" respawn="true" />

</launch>
